src/build-data/os/android.txt

```diff
@@ -18,4 +18,5 @@ getauxval
 sockets
 threads
 filesystem
+sysconf
```

Comment:
sysconf function is part of POSIX.1-2001 so these can be removed and go off `defined(BOTAN_TARGET_OS_HAS_POSIX1)`